sentry-ruby/lib/sentry/propagation_context.rb

```diff
@@ -61,6 +109,8 @@ def initialize(scope, env = nil)
                 Baggage.new({})
               end
 
+            @sample_rand = self.class.extract_sample_rand_from_baggage(@baggage, @trace_id) ||
```

Comment:
missing after the `||` here or the `||` is not necessary?

Reply:
OH WOW Ruby can surprise me even after 18 years 🤣 I cut the end after `||` and forgot to remove it